src/components/CardCover.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="card-cover-wrap" :class="{ rounded }">
+    <slot classes="card-cover">
+      <div class="card-cover card-cover--light" :style="styles.light" />
```

Comment:
Why not using the `ImageAsset` component instead? so we could load the images on demand based on the prefers-color-scheme instead of loading both as backgrounds

Reply:
Good question. We are using `background-size: cover` to position the images, but I guess I could refactor that to `object-fit`. Stay tuned.

Reply:
Done.